### ISSUE TITLE
help

### ISSUE THREAD

**Issue by missiontn** [Avinash9502/IMEI-tracker#1]
how to launch this program in kali linux 2020.3

**Comment by BonnY0** [comment]
Yeah how to build and run this, cuz it is and old java code and there is no tutorial anywhere on java builds in linux.Help someone pls?

**Comment by JOoSab** [comment]
how to launch this program in kali linux

**Comment by g85606572** [comment]
Helo..

**Comment by g85606572** [comment]
database password Err..

anyone can Help?

**Comment by kalien-parrien** [comment]
> Helo..

Helo

**Comment by g85606572** [comment]
@kalien-parrien 

Help me 
To understand this program...

**Comment by 7xhavok** [comment]
dude add instructions please

**Comment by MADDY312** [comment]
does this actually gives gps location?

**Comment by IVAR23-NITRO** [comment]
Bro just give guide how to run the code

**Comment by vinkeaxeltiemtore** [comment]
> Bro donne juste un guide qui guide comment exécuter le code


sur kali linux, java -jar nomdufichier.jar

**Comment by taintunde** [comment]
database password needed bro

**Comment by taintunde** [comment]
> how to launch this program in kali linux

install netbeans
